### PR TITLE
Added 2 farms again

### PR DESCRIPTION
**Pull request by FarkasKBence**
#### Summary

 ```SUMMARY: Content Added 2 farms```  

#### Purpose of change
The code is from #28358 
There was only one farm in the game. 
#### Describe the solution
Added 2 farms. Now there's 3.

#### Describe alternatives you've considered

Not changing it. It's not an important update.

#### Screenshots
farm_stills

farm_2side

### CODE REVIEW

**Comment by I-am-Erk** [comment]
Thanks for reposting. Sorry for all the fuss. I'll review this as soon as I can load it into my game and test it out.

**Comment by ralreegorganon** [comment]
Repeating the comment from #28427 as it applies here as well:

In general, the minimum occurrences for any overmap special that is part of the base game should be zero.

**Comment by AMurkin** [comment]
` farm_2side.json` and `farm_stills.json` files should go into `data/json/mapgen` or one of its sub directories.

File: farm_2side.json [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        "P": { "item": "mail", "chance": 30, "repeat": [ 2, 5 ] },
+        "l": [
+          { "item": "tools_earthworking", "chance": 65 },
+          { "item": "mechanics", "chance": 40 },
+          { "item": "home_hw", "chance": 50 },
+          { "item": "tools_common", "chance": 50 }
+        ],
+        "u": [
+          { "item": "kitchen", "chance": 70 },
+          { "item": "cannedfood", "chance": 65 },
```
Comment (I-am-Erk) [inline farm_2side.json]: Overall I'm working on increasing some types of item spawns significantly, to indicate the game start time as only a few days after the end of the world. I'd suggest you could add some repeats to several of these thngs like fridge, bedroom, and cannedfood if you want.

**Comment by LovamKicsiGazsii** [comment]
I'll try to increase the item spawns. I am not adding roofs for now, but I will in the next one. 
Also is there a simple way to move these files to ```data/json/mapgen``` ?
Or do I have to revert ```farm_2side_json``` and ```farm_stills``` and add them again to the right folder?

File: data/json/overmap_terrain.json [hunk 2]
```diff
@@ -916,6 +916,286 @@
...
+  {
+    "type": "overmap_terrain",
+    "id": "farm_stills_12",
+    "name": "grape farm",
+    "sym": 35,
+    "color": "i_brown",
+    "see_cost": 5,
+    "extras": "field",
+    "mondensity": 2
+  },
```
Comment (ZhilkinSerg) [inline data/json/overmap_terrain.json]:
```suggestion
  },
```

**Pull request merged**